### DISCUSSION TITLE
VSCode insiders behaves weirdly when connected to VPN

### DISCUSSION THREAD

**Discussion by sanket-bhalerao** [microsoft/vscode-discussions#25] [Extension Development QnA] [answered]
This weird behavior is on my company laptop where I need to connect to VPN.
following behaviors observed so far.
* Extension does not update
* After a while Extension panel stops working. If you try to search for an extension just a loader will be visible forever. Clearing the search will show a blank extension panel.
* In Git difference view, there are missing icons
![image](https://user-images.githubusercontent.com/9264813/172290382-fa4122b8-5877-42f1-bae6-77f7992f24ef.png)

* Sometimes Git diff is not visible

If I disconnect from the VPN things work fine, however, as I am working on a company laptop I need to be constantly connected to VPN. Making me not use Insiders build :( .

This behavior happens on the VSCode insiders build only. Stable VSCode works fine.
Facing this for the last couple of months

**Comment by isidorn** [comment, marked as answer]
Sounds like a bug. I suggest to open a new issue in the vscode repository https://github.com/microsoft/vscode/issues

**Reply by sanket-bhalerao** [reply]
@isidorn I have a bug opened for this [buggy bugggity bug](https://github.com/microsoft/vscode/issues/144153), but it's been a while, and everything there is cold. Wanted to check if there is anyone else facing something similar.

**Reply by burkeholland** [reply]
Can we close this item? Looks like it was answered in your issue there (by you 😀)

**Reply by sanket-bhalerao** [reply]
yes @burkeholland.
how to close it BTW?
